main/src/main/java/gov/nasa/jpf/vm/NativeStackFrame.java

```diff
@@ -101,78 +91,78 @@ public void setReturnValue(Object r){
     ret = r;
   }
 
-  public void clearReturnValue() {
+  public void clearReturnValue(){
     ret = null;
     retAttr = null;
   }
 
-  public Object getReturnValue() {
+  public Object getReturnValue(){
     return ret;
   }
 
-  public Object getReturnAttr() {
+  public Object getReturnAttr(){
     return retAttr;
   }
 
-  public Object[] getArguments() {
+  public Object[] getArguments(){
     return args;
   }
 
   @Override
-  public void markThreadRoots (Heap heap, int tid) {
+  public void markThreadRoots(Heap heap, int tid){
     // what if some listener creates a CG post-EXECUTENATIVE or pre-NATIVERETURN?
     // and the native method returned an object?
     // on the other hand, we have to make sure we don't mark a return value from
     // a previous transition
 
-    if (pc instanceof NATIVERETURN){
-      if (ret != null && ret instanceof Integer && mi.isReferenceReturnType()) {
-        int ref = ((Integer) ret).intValue();
+    if (instructionToExecute instanceof NATIVERETURN) {
+      if (ret != null && ret instanceof Integer && methodInfo.isReferenceReturnType()) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [These nested if statements could be combined](https://www.codacy.com/app/grzesuav/gjpf-core/file/1824888150/issues/source?bid=2176522&fileBranchId=3040268#l119)